mmv1/products/dataprocgdc/ServiceInstance.yaml

```
@@ -0,0 +1,123 @@
+    output: true
+  - name: requestedState
+    type: String
+    description: "Output only. The intended state to which the service instance is reconciling.
+      \n Possible values:\n STATE_UNSPECIFIED\nCREATING\nACTIVE\nDISCONNECTED\nDELETING\nSTOPPING\nSTOPPED\nSTARTING\nUPDATING\nFAILED"
+    output: true
+  - name: state
+    type: String
+    description: "Output only. The current state. \n Possible values:\n STATE_UNSPECIFIED\nCREATING\nACTIVE\nDISCONNECTED\nDELETING\nSTOPPING\nSTOPPED\nSTARTING\nUPDATING\nFAILED"
+    output: true
+  - name: reconciling
+    type: Boolean
+    description: "Output only. Whether the service instance is currently reconciling.\nTrue
+      if the current state of the resource does not match the\nintended state, and the
+      system is working to reconcile them, whether\nor not the change was user initiated.\nRequired
+      by aip.dev/128#reconciliation "
+    output: true
+  - name: labels
+    type: KeyValueLabels
+    output:
+    api_name:
+    description: "Optional. The labels to associate with this service instance. Labels
+      may be used for\nfiltering and billing tracking. "
+    min_version:
+    ignore_write:
+    update_verb:
+    update_url:
+    immutable:
+  - name: sparkServiceInstanceConfig
+    type: NestedObject
+    properties: []
+    description: 'Spark-specific service instance configuration. '
+  - name: stateMessage
+    type: String
+    description: 'Output only. A message explaining the current state. '
+    output: true
+  - name: serviceAccount
+    type: String
+    description: 'Optional. Requested service account to associate with ServiceInstance. '
+  - name: effectiveServiceAccount
+    type: String
+    description: "Output only. Effective service account associated with ServiceInstance.
+      This will be\nthe service_account if specified. Otherwise, it will be an automatically\ncreated
+      per-resource P4SA that also automatically has Fleet Workload\nIdentity bindings
+      applied. "
+    output: true
+parameters:
+  - name: location
+    type: String
+    description: 'Resource ID segment making up resource `name`. It identifies the resource
+      within its parent collection as described in https://google.aip.dev/122. See documentation
+      for resource type `dataprocgdc.googleapis.com/SparkApplication`. '
+    url_param_only: true
+    required: true
+    immutable: true
+  - name: serviceInstanceId
+    type: String
+    description: "Required. Id of the requesting object\nIf auto-generating Id server-side,
+      remove this field and\nservice_instance_id from the method_signature of Create
+      RPC "
+    url_param_only: true
+    required: true
+    immutable: true
+async:
+  actions: ['create', 'delete']
+  type: OpAsync
+  operation:
+    path: name
+    base_url: "{{op_id}}"
+    wait_ms: 1000
+    timeouts:
+  result:
+    path: response
+    resource_inside_response: false
+  error:
+    path: error
+    message: message
```

Most of these are unused (and `resource_inside_response: false` is the default) - see https://github.com/hashicorp/terraform-provider-google/issues/17098 for more details.

```suggestion
async:
  actions: ['create', 'delete']
  type: OpAsync
  operation:
    base_url: "{{op_id}}"
```

---

Done